rasa_nlu_models/common/mdr_hri_behaviours_models/nlu/checkpoint

```diff
@@ -0,0 +1,2 @@
+model_checkpoint_path: "/home/patrick/rasa/models/current/nlu/intent_classifier_tensorflow_embedding.ckpt"
```

Comment:
Is this file necessary?

Reply:
Most likely no. As I mentioned before, it is automatically generated when training the model. Should I remove them?. @argenos